xml/System.IO.Compression/GZipStream.xml

```diff
@@ -1470,13 +1577,25 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="3" FrameworkAlternate="netcore-1.0;netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-1.3;netstandard-1.4;netstandard-1.6;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="array">To be added.</param>
-        <param name="offset">To be added.</param>
-        <param name="count">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="array">The buffer to write data from.</param>
```

Comment:
"Write data from" is a bit confusing since generally we **read** data **from** an array, but **write** **to** an array.

Maybe these should follow how DeflateStream is documented:
`The buffer that contains the data to compress.`

https://docs.microsoft.com/en-us/dotnet/api/system.io.compression.deflatestream.write?view=netstandard-2.1

Applies to a some of the other APIs/params as well.

Reply:
I see what you mean. We read from this array, we write into the stream. I'll fix it in all 3 files. Thanks for the suggestion.

Reply:
To be clear, I am suggesting lets be consistent with the existing DeflateStream API docs for a lot of these APIs (not just this one instance).

Reply:
Addressed. I also modified the summaries for all WriteAsync methods, since I was describing the underlying stream as a normal stream, not as one that gets compressed.